packages/editor/src/components/post-format/index.js

```diff
@@ -55,57 +79,30 @@ function PostFormat( {
 						{ __( 'Post Format' ) }
 					</label>
 					<SelectControl
-						value={ postFormat }
-						onChange={ ( format ) => onUpdatePostFormat( format ) }
+						value={ currentFormat }
+						onChange={ ( format ) => updatePostFormat( format ) }
 						id={ postFormatSelectorId }
-						options={ formats.map( ( format ) => ( {
-							label: format.caption,
-							value: format.id,
+						options={ listedFormats.map( ( format ) => ( {
+							label: POST_FORMAT_TITLES[ format ],
+							value: format,
 						} ) ) }
 					/>
 				</div>
 
-				{ suggestion && suggestion.id !== postFormat && (
+				{ suggestedFormat && (
```

Comment:
Similar to the previous comment, the code changes `suggestedFormat` to be validated against `supportedFormats` which logically causes an issue when that values doesn't exist in `POST_FORMAT_TITLES`.

Previously the value was sourced from `POST_FORMATS`.